requirements.txt

```diff
@@ -4,7 +4,7 @@ matplotlib>=3.9.1,<3.10.0
 numpy>=2.1.0,<2.2.0
 photutils>=1.13.0,<2.1.0
 poppy>=1.0.0
-pysiaf>=0.22.0,<0.24.0
+pysiaf>=0.23.3,<0.24.1
```

Comment:
Should this be "<=0.24.1"? 

I would think that the current most recent released version 0.24.1 is usable here, yes?

Reply:
Yes, good catch. Fixed.